test/pageobjects/googleCloudPlatform.page.js

```diff
@@ -0,0 +1,126 @@
+const Page = require('./page');
+
+
+class Pastebin extends Page {
+
+   get searchCalculator(){
+    return $(`//input[@name='q']`)
```

Comment:
`'//input[@name="q"]'`

Please check the whole PR for using backticks only for string interpolation.